common/crypto_utils/setup.py

```diff
@@ -63,25 +63,26 @@
 
 verify_report_include_dirs = [
     os.path.join(tcf_root_dir, 'common/cpp'),
+    os.path.join(tcf_root_dir, 'common/cpp/crypto'),
+    os.path.join(tcf_root_dir, 'common/cpp/packages/parson'),
+    os.path.join(tcf_root_dir, 'common/cpp/packages/base64'),
     os.path.join(tcf_root_dir, 'common/cpp/verify_ias_report'),
-]
+] + openssl_include_dirs
 
 library_dirs = [
     os.path.join(tcf_root_dir, "common/cpp/build"),
 ] + openssl_lib_dirs
 
 libraries = [
+    'uavalon-verify-ias-report',
+    'uavalon-crypto',
```

Comment:
Just curious. Did the order of adding library matter ?

Reply:
yes changing order fixed the undefined symbol error in libuavalon-verify-ias-report.